Projects/Server/Timer/Timer.TimerWheel.cs

```diff
@@ -214,18 +224,19 @@ private static void AddTimer(Timer timer, long delay)
                     }
                 }
 
+                timer.Next = Core.Now + timer.Delay;
```

Comment:
Note that `Next` is now assigned when the timer is added. It no longer erroneously reflects the Next tick based on when the timer is added.